.github/workflows/e2e-tests.yaml

```diff
@@ -16,15 +16,14 @@ concurrency:
   group: ${{ github.repository }}-concurrency-for-e2e
   cancel-in-progress: false
 
-env:
-  IS_PR: ${{ github.event_name == 'pull_request' }}
-  PR_IS_APPROVED: ${{ github.event.review.state == 'APPROVED' }}
-  PR_IS_NOT_IN_DRAFT: ${{ !github.event.pull_request.draft }}
-
 jobs:
   e2e-tests:
     name: Run E2E Tests
-    if: ('$IS_PR' && ( '$PR_IS_APPROVED' || '$PR_IS_NOT_IN_DRAFT' )) || '$IS_PR' == 'false'
+    if: |
```

Comment:
MODIFICATION: Isolate pull_request and pull_request_review event triggers and update non-pull_request trigger to account for when pull_request_review event is activated/triggered.